developer-workflow/c-api.rst

```diff
@@ -101,12 +101,12 @@ C API Tests
 Tests for the public C API live in the ``_testcapi`` module.
 Functions named ``test_*`` are used as tests directly.
 Tests that need Python code (or are just easier to partially write in Python)
-live in ``Lib/test``, mainly in :file:`Lib/test/test_capi.py`.
+live in ``Lib/test``, mainly in :cpy-file:`Lib/test/test_capi`.
```

Comment:
We could also link things that aren't already `:file:`, like `Lib/test`:

```suggestion
live in :cpy-file:`Lib/test`, mainly in :cpy-file:`Lib/test/test_capi`.
```

There's a lot more that could be linked on this page too, especially the first few sections.

But let's keep the scope of this PR to replacing the old `:file:` with the new `:cpy-file:`.

Reply:
Agreed on keeping the PR focused.  We should also try to create links only when/where they are useful, but there are definitely more places where `:cpy-file:` can be used.

Reply:
That SGTM